src/components/Profile/components/PersonalInfoList/components/UpdatePhoneDialog/index.js

```diff
@@ -0,0 +1,126 @@
+import React, { useState } from 'react';
+import MaskedInput from 'react-text-mask';
+import EditIcon from '@material-ui/icons/Edit';
+import {
+  IconButton,
+  FormControl,
+  Input,
+  InputLabel,
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+  DialogTitle,
+} from '@material-ui/core';
+import userService from 'services/user';
+import user from 'services/user';
+import GordonSnackbar from 'components/Snackbar';
+import './index.css';
+
+const UpdatePhone = () => {
+  const [open, setOpen] = React.useState(false);
+  const [mobilePhoneNumber, setMobilePhoneNumber] = useState();
+  const [snackbar, setSnackbar] = useState({ message: '', severity: null, open: false });
+
+  const handleClickOpen = () => {
```

Comment:
I feel like a function like this isn't necessary. It just has one line. What if in the buttons that call this in their onClick, you had them just execute `setOpen(false)` and save yourself a function?

Reply:
Same thing with `handleClose`.